ranger-gcs-permission-check-service/src/main/java/com/google/cloud/hadoop/ranger/gcs/permissionCheckService/utilities/Config.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *  * Copyright 2020 Google Inc. All Rights Reserved.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *    http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.google.cloud.hadoop.ranger.gcs.permissionCheckService.utilities;
+
+public class Config {
+    // IP and port to host the permission check service.
+    private String hostIp;
+    private int hostPort;
+
+    public String getHostIp() {
+        return hostIp;
+    }
+
+    public void setHostIp(String hostIp) {
```

Comment:
do you need setters or just a constructor? Shouldn't config classes be immutable?
Also, if you are interested: https://github.com/google/auto/blob/master/value/userguide/index.md